_dasldemos/code-review-guidance.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Code Review Guidance
+main_author: Logan Wallace
+primary_reviewers:
+---
+
+See below for guidance on code review. 
+
+[Fred Hutch DaSL Code Review](https://hutchdatascience.org/code_review/)
+
+[WILDS Code Review Article](https://getwilds.org/guide/codereview.html)
```

Comment:
Looks awesome! Could we just update the text to "Fred Hutch DaSL Code Review" and "WILDS Code Review Guidance"? Super minor, but just want to be clear about what each resource is.

Reply:
Hi @tefirman, sounds like a plan to me! Will do this now. Good thought.